tendrl-gluster-integration.spec

```diff
@@ -15,6 +15,7 @@ BuildRequires: python-mock
 Requires: tendrl-commons
 Requires: systemd
 Requires: gstatus
```

Comment:
We are not using gstatus anymore right? So better to remove it from the spec file or we are doing this as part of any other patch?